config/presets/smart_access_brands.json.erb

```diff
@@ -0,0 +1,13 @@
+{
+  "title": "Example SMART Access Brands Bundle",
+  "id": "smart_access_brands_bundle_example",
+  "test_suite_id": "smart_access_brands",
+  "inputs": [
+    {
+      "name": "smart_access_brands_bundle_url",
+      "value": "<%= Inferno::Application['base_url'] %>/custom/smart_access_brands/examples/smart_access_brands_example.json",
```

Comment:
Did we create this example?

I think it might be a good idea to have the examples provided by the guide itself:

* https://hl7.org/fhir/smart-app-launch/Bundle-example1.json
* https://hl7.org/fhir/smart-app-launch/Bundle-example2.json
* https://hl7.org/fhir/smart-app-launch/Bundle-example3.json
* https://hl7.org/fhir/smart-app-launch/Bundle-example4.json 

In the preset, make sure to set the "Endpoint Availability Success Rate (required) *" to "none" for these, as we want to demonstrate that the examples provided are indeed correct (as long as there isn't an expectation that the endpoints are currently accessible).

As far as including the existing one that we made as well, I'm ambivalent.  Does it add anything on top of the examples provided in the IG?  If yes, then I think we should just make sure to title the presets in a way that makes sense (this is an example we came up with... then there are the examples from the IG itself).

Reply:
Hmmmm, I also get this validation error for one of the examples though:  ```Organization.extension[1].extension[1].value.ofType(url): Example URLs are not allowed in this context ```

Hmmm, I think we should just shut that off in the validator.  Checking the format of the URL isn't really relevant to us, as we let the tester choose to actually try to resolve the URLs.

I'll add a comment to the right spot to turn that off in the validator.

Reply:
In terms of whether or not to keep the existing one we made, the only thing it really adds on top of the other examples is that Inferno can host the example endpoints provided in this Bundle and thus do availability validation, whereas these IG examples if we set the success rate input to "None", then there will be no examples that do availability validation. However, I am not sure how much that really matters, so I will leave it up to you whether or not we get rid of it

Reply:
Good point, let's leave it.